src/main/java/com/google/step/servlets/UserPostUrlServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.step.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles usr upload url. */
+@WebServlet("/api/user-post-url/*")
+public class UserPostUrlServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id;
+    try {
+      String idString = request.getPathInfo().substring(1);
```

Comment:
might need to check if `request.getPathInfo()` is null if user goes to url `/api/user-post-url`
![Screenshot 2020-07-09 at 1 16 02 PM](https://user-images.githubusercontent.com/39252823/86999916-5d4a2300-c1e6-11ea-8874-21c7e4075b09.png)